lib/scientist/experiment.rb

```diff
@@ -318,4 +326,14 @@ def generate_result(name)
     control = observations.detect { |o| o.name == name }
     Scientist::Result.new(self, observations, control)
   end
+
+  private
+
+  def marshalize(block)
```

Comment:
Can you add a comment here explaining why we want to make everything marshalable?

Reply:
Good call. Added. 👌